src/current/molt/molt-overview.md

```diff
@@ -0,0 +1,58 @@
+---
+title: MOLT Overview
+summary: Learn how to use the MOLT tools to migrate to CockroachDB.
+toc: true
+docs_area: migrate
+---
+
+This page describes the MOLT (Migrate Off Legacy Technology) tools. For more information about migrating to CockroachDB, see [Migration Overview]({% link {{site.current_cloud_version}}/migration-overview.md %}).
+
+The MOLT tools can be used when:
+
+- Converting a schema for compatibility with CockroachDB.
+- Loading test and production data into CockroachDB.
+- Validating queries on CockroachDB.
+- Performing a cutover to finalize the migration to CockroachDB.
+
+## MOLT tools
+
+<img src="{{ 'images/molt/molt_tools.svg' | relative_url }}" alt="MOLT tooling overview" style="max-width:100%" />
+
+|                        Tool                       |          Supported databases          |                   Usage                   |                                      Release status                                      |
+|---------------------------------------------------|---------------------------------------|-------------------------------------------|------------------------------------------------------------------------------------------|
+| [Schema Conversion Tool](#schema-conversion-tool) | PostgreSQL, MySQL, Oracle, SQL Server | Schema conversion                         | GA                                                                                       |
+| [Fetch](#fetch)                                   | PostgreSQL, MySQL, CockroachDB        | Initial data load; continuous replication | GA                                                                                       |
+| [Verify](#verify)                                 | PostgreSQL, MySQL, CockroachDB        | Data validation                           | [Preview]({% link {{site.current_cloud_version}}/cockroachdb-feature-availability.md %}) |
+| [Live Migration Service](#live-migration-service) | PostgreSQL, MySQL, CockroachDB        | Consistent cutover                        | [Preview]({% link {{site.current_cloud_version}}/cockroachdb-feature-availability.md %}) |
+
+### Schema Conversion Tool
+
+The [MOLT Schema Conversion Tool]({% link cockroachcloud/migrations-page.md %}) [converts a source database schema]({% link {{site.current_cloud_version}}/migration-overview.md %}#convert-the-schema) to a CockroachDB-compatible schema. The supported Schema Conversion Tool sources are PostgreSQL, MySQL, Oracle, and SQL Server.
```

Comment:
Here we say "supported Schema Conversion Tool _sources_".  I'm wondering if this means that we should relabel the column in the table above from "Supported Databases" to "Supported Sources".  I personally find the latter more intuitive.

Reply:
I agree we should be consistent. To bikeshed here for a minute, we sometimes talk about sources, databases, and dialects. I personally like databases because thats what they are but sources are more general (perhaps overtime S3 is a source for example?) and dialects feels more clinical. I dont have strong preferences but i do think we should pick one and be consistent